clients/java/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'java'
```

Comment:
For others reviewing this. Docs on the "java" plugin: https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_project_layout